app/components/forms/orders/attendee-list.js

```diff
@@ -74,8 +74,9 @@ export default class AttendeeList extends Component {
         field.transName = field.name;
       }
 
-      return field.form;
-    });
+      return true;
```

Comment:
i think this line of code is redundant. please check thanks!

Reply:
But something needs to be returned. I am more concerned about why `true`? And how the sorting works.

Reply:
The sorting process works in analogue to the sorting in the `allFields` method of `order-form.js`. That's also where I got that filtering from, even though the `return true` was pretty useless here. However, as Norbert mentioned, something needs to be retuned. I therefore now switched that `filer` to a `map` function. This returns the `field` objects that got modified in the function, in order to then have them accessible for the sorting. What do you think of that solution?

Reply:
This is GTG